tests/trigger-record-logging.unit.test.ts

```
@@ -0,0 +1,260 @@
+import { describe, expect, it } from 'vitest'
+
+import {
+  getAppVersionTriggerRecordLogMetadata,
+  getAppTriggerRecordLogMetadata,
```

ESLint

[error] Expected "getAppTriggerRecordLogMetadata" to come before "getAppVersionTriggerRecordLogMetadata".

(perfectionist/sort-named-imports)